packages/squiggle-lang/src/errors/messages.ts

```diff
@@ -177,3 +177,13 @@ export class REAmbiguous extends ErrorMessage {
     return `Ambiguous Error: ${this.message}`;
   }
 }
+
+export class REBuildError extends ErrorMessage {
```

Comment:
Not sure if this deserves a separate error class, since it will happen on builds and tests would never pass. I'd probably just throw an untyped Error, like we do in a few other cases:

```
$ rg 'throw new Error' src/library/
src/library/registry/core.ts
137:      throw new Error(`Function ${fnName} has no definitions`);

src/library/registry/frTypes.ts
471:      throw new Error("Unable to pack null value");
489:        throw new Error("Unable to pack null value");

src/library/registry/squiggleDefinition.ts
25:    throw new Error(`Stdlib code ${code} is invalid`);
```

Even if your intention was to reuse it for user-defined functions when we add types there, it'd be a compile-time error which belongs to a different type hierarchy (`SqCompileError` -> `ICompileError`, there's no `ErrorMessage` there).